src/pjpersist/datamanager.py

```diff
@@ -442,25 +460,21 @@ def __init__(self, conn, root_table=None):
         self._loaded_objects = {}
         self._inserted_objects = {}
         self._removed_objects = {}
+        self._dirty = False
+
+        self._readonly = None
```

Comment:
why not default to False? why make this optional?

Reply:
If it is not set (i.e. is `None`), it will not be explicitly set database session on connection, meaning the default value will be used. I think this is safer then always force-setting readonly to session variable to False by default - it is better to keep the server default in this case.